scala3/guides.md

```diff
@@ -18,6 +18,11 @@ guides:
     icon: birthday-cake
     url: "/scala3/guides/tasty-overview.html"
     description: "An overview over the TASTy format aimed at end-users of the Scala language."
+  - title: Scaladoc
+    by: Krzysztof Romanowski, Aleksander Boruch-Gruszeski, Andrzej Ratajczak
```

Comment:
Please let me know if I missed someone

Reply:
```suggestion
    by: Krzysztof Romanowski, Aleksander Boruch-Gruszecki, Andrzej Ratajczak
```

@KacperFKorban @pikinier20, have you contributed to these docs?

Reply:
I think that I haven't...yet

Reply:
`Aleksander Boruch-Gruszecki`